sdk/src/main/java/io/opentelemetry/sdk/metrics/aggregator/LongLastValueAggregator.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.metrics.aggregator;
+
+import io.opentelemetry.sdk.metrics.data.MetricData.LongPoint;
+import io.opentelemetry.sdk.metrics.data.MetricData.Point;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicLong;
+
+public final class LongLastValueAggregator extends AbstractAggregator {
+
+  private static final long DEFAULT_VALUE = 0L;
```

Comment:
I don't think this is a good default. I actually think we should return a null Point if nothing has been recorded, rather than returning a possibly misleading `0`.

Reply:
Documented in the description this limitation.

Reply:
Done.